src/components/AbuseReportForm/AbuseReportForm.tsx

```diff
@@ -135,47 +135,48 @@ export const AbuseReportForm: React.FC<{
         response: '',
     };
     const [errors, setErrors] = useState(defaultErrorTexts);
-    const onSubmit = async () => {
+    const [successMessage, setSuccessMessage] = useState();
+    const onSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+
         const { categoryId, reason, email } = formVariables;
+
         // Reset error messages
         setErrors(defaultErrorTexts);
+
         // Error validation
         if (!categoryId) {
             setErrors({
                 ...errors,
                 categoryId: 'You must select a category before submitting',
             });
+
+            return false;
         }
         const response = await reportAbuse({
             categoryId,
             reason,
             email,
             commentId,
         });
-        if (!response.ok) {
+        if (response.status !== 'ok') {
+            // Fallback to errors returned from the API
             setErrors({ ...errors, response: response.message });
         } else {
-            toggleSetShowForm();
-            // TODO: display sucess?
+            setSuccessMessage('Report submitted');
         }
     };
 
     const labelStylesClass = labelStyles(pillar);
     return (
         <div aria-modal="true" ref={modalRef}>
             <form className={formWrapper} onSubmit={onSubmit}>
-                {errors.response && (
-                    <span className={errorMessageStyles}>
-                        {errors.response}
-                    </span>
-                )}
-
                 <div className={inputWrapper}>
                     <label className={labelStylesClass} htmlFor="category">
                         Category
                     </label>
                     <select
-                        name="categoryId"
+                        name="category"
```

Comment:
This change is breaking some of the accessiblity features. I agree though that `categoryId` is a bad name but if we change it we should carry that through the code

Reply:
> This change is breaking some of the accessiblity features. I agree though that `categoryId` is a bad name but if we change it we should carry that through the code

I actually changed it because it didn’t match the label for attribute previously... I’ll double check.

Reply:
In fact, I’m fairly sure this is correct and I noticed because clicking the label didn’t select the field before this change!

Reply:
I made [a change](https://github.com/guardian/discussion-rendering/pull/249/commits/edb4ac9e492349a8d415dbad769f0ca4c4ea82fe) that I think fixes this. The code to override the tabbing was expecting the name to be `categoryId` but I changed it to `category`.

Reply:
Ah perfect thanks!